### PR TITLE
TU - 11 -display in profile page -UI

### PR DESCRIPTION
**Pull request by TeamUp3Batch**
TU - 11 -display in profile page changes

### CODE REVIEW

File: src/pages/Header/Header.jsx [hunk 1]
```diff
@@ -40,6 +42,15 @@ const Header = () => {
       console.log(error);
```
Comment (anithaamarnath) [inline src/pages/Header/Header.jsx:42]: Remove the console.log , thanks

File: yammydash.code-workspace [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+	"folders": [
```
Comment (anithaamarnath) [inline yammydash.code-workspace:2]: I have no idea about this file

File: how [hunk 3]
```diff
@@ -0,0 +1,504 @@
+[33mcommit 326407dc13477995ad8d8686c1e625707bcbd266[m[33m ([m[1;36mHEAD -> [m[1;32mnaser-frontend[m[33m, [m[1;31morigin/main[m[33m, [m[1;31morigin/HEAD[m[33m, [m[1;32mmain[m[33m)[m
```
Comment (anithaamarnath) [inline how:1]: Remove this file, would be great, Thank u

**Review by anithaamarnath — CHANGES_REQUESTED** [review]
For me, it looks great, once file name consistency ViewAccount folder(Profile) naming conversion or vice versa

File: package.json [hunk 4]
```diff
@@ -4,16 +4,26 @@
   "private": true,
   "dependencies": {
     "@auth0/auth0-react": "^2.2.1",
+    "@chakra-ui/react": "^2.8.1",
```
Comment (anithaamarnath) [inline package.json:7]: If we not implement chakra-ui, we can uninstall

File: src/pages/ViewAccount/Profile.jsx [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+          <br />
+          <span>Email: </span> {loggedInUser.email}
+          
+        </TabPanel>
+        <TabPanel value={value} index={1} dir={theme.direction}>
+          <br />
+          <h3>Reset Your Password</h3>
+          <p>Enter your email address to reset your password. <br />
+          You may need to check your spam folder or unblock <br />
+          support@yummydash.com
```
Comment (anithaamarnath) [inline src/pages/ViewAccount/Profile.jsx:108]: Remove this information because we not gonna support for now
Thanks

**Review by anithaamarnath — APPROVED** [review]

**Review by kavithashanmugan — APPROVED** [review]

**Pull request merged**